app/app/src/main/res/drawable/circle.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file was removed in a previous merge because it is no longer used in the current version of our app. It used to belong to the EEG animation, back when the animation relied on hard-coded drawable files represented as ImageViews. It was a sad time. So happy we have the custom view now <3